tests/jest/components/tab/SearchInput.spec.js

```diff
@@ -176,15 +176,22 @@ describe('SearchInput.vue', () => {
 				await wrapper.setData({
 					searchResults: [],
 				})
-				const multiSelectCont = wrapper.find(multiSelectContentSelector)
+				const multiSelectCont = wrapper.find(workpackagesListSelector)
 				expect(multiSelectCont).toMatchSnapshot()
 			})
 			it('should display correct options list of search results', async () => {
+				jest.spyOn(axios, 'get')
+					.mockImplementationOnce(() => Promise.resolve({
+						status: 200,
+						data: [],
+					}))
+				wrapper = mountSearchInput({ id: 1234, name: 'file.txt' })
+				const inputField = wrapper.find(inputSelector)
+				await inputField.setValue(' ')
 				await wrapper.setData({
-					fileInfo: { id: 1234 },
 					searchResults: workPackagesSearchResponse,
 				})
-				const multiSelectContent = wrapper.find(multiSelectContentSelector)
+				const multiSelectContent = wrapper.find(workpackagesListSelector)
```

Comment:
It's not multiseletc anymore so can we change the name to something more appropriate 
```suggestion
				const ncSelectContent = wrapper.find(workpackagesListSelector)
```